### ISSUE TITLE
[remote,rootless] podman build: could not bind mount \"/sys/kernel/debug/tracing\", skipping: permission denied

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#8778]
Fallout from #8479 (setting log level to `warn`):

In window 1:
```console
$ bin/podman system service --timeout=0
```
In window 2:
```console
$ printf "FROM quay.io/libpod/testimage:20200929\nRUN true\n" | bin/podman-remote build -t foo -
STEP 1: FROM quay.io/libpod/testimage:20200929
STEP 2: RUN true
time="2020-12-18T09:02:46-07:00" level=warning msg="could not bind mount \"/sys/kernel/debug/tracing\", skipping: permission denied"
STEP 3: COMMIT foo
...
```
I'm disappointed that my system tests didn't catch this.

**Comment by rhatdan** [comment]
Are you building with BUILDAH_ISOLATION=chroot?

I can not recreate this.  I guess this is something about your environment setup.
Looks like your host has "/sys/kernel/debug/tracing" as a mount point?

mount | grep /sys/kernel/debug/tracing

**Comment by edsantiago** [comment]
No special environment: `env|egrep 'BUI|ISO'` shows no hits. I have no idea what `tracefs` is or where it came from or if it's normal or not.

```console
tracefs on /sys/kernel/debug/tracing type tracefs (rw,nosuid,nodev,noexec,relatime,seclabel)
# journalctl -b|grep tracefs -> no results
```

I'm still seeing this on master @ 076f77b964329c830664b4d6fa3234f608f07591

**Comment by rhatdan** [comment]
If you run lsof it causes this file system to get mounted.  I have a reproducer.

**Comment by edsantiago** [comment]
Confirmed fixed by #8978
